anchor-reference-server/src/test/kotlin/org/stellar/anchor/server/AnchorReferenceServerIntegrationTest.kt

```diff
@@ -36,6 +39,51 @@ class AnchorReferenceServerIntegrationTest {
   @Autowired lateinit var restTemplate: TestRestTemplate
   @Autowired lateinit var quoteRepo: QuoteRepo
 
+  @Test
+  fun getFee() {
+    val result =
+      restGetFee(
+        GetFeeRequest.builder()
+          .sendAmount("10")
+          .sendAsset("USDC")
+          .receiveAsset("USDC")
+          .senderId("sender_id")
+          .receiverId("receiver_id")
+          .build()
+      )
+    print(result.body)
```

Comment:
We should remove this print statement right?

Reply:
Removed.